target/linux/malta/config-5.10

```diff
@@ -0,0 +1,340 @@
+CONFIG_ARCH_32BIT_OFF_T=y
```

Comment:
This is interesting. What does it do?

Reply:
It's automatically set from [arch/mips/Kconfig](https://github.com/torvalds/linux/blob/f40ddce88593482919761f74910f42f4b84c004b/arch/mips/Kconfig#L5). AFAICT it's used on legacy archs to [support 32-bit userspace offsets](https://github.com/torvalds/linux/blob/0653161f0faca68b77b3f36fb4b4b9b8b07050e5/arch/Kconfig#L320). This instance is simply copied from config-5.4.